Cython/Debugger/Cygdb.py

```diff
@@ -48,13 +48,19 @@ def make_command_file(path_to_debug_info, prefix_code='', no_import=False):
             try:
                 # Activate virtualenv, if we were launched from one
                 import os
+                import sys
+                import site
```

Comment:
Since we're only using these in the case of a virtualenv, I suggest moving the imports under the `if virtualenv` condition.